apps/twr_aloha/pkg.yml

```diff
@@ -43,8 +43,8 @@ pkg.deps:
     - "@mcuboot/boot/bootutil"
     - "@apache-mynewt-core/kernel/os"
     - "@apache-mynewt-core/mgmt/imgmgr"
-    - "@apache-mynewt-core/mgmt/newtmgr"
-    - "@apache-mynewt-core/mgmt/newtmgr/transport/nmgr_shell"
+    - "@apache-mynewt-core/mgmt/smp"
+    - "@apache-mynewt-core/mgmt/smp/transport/smp_shell"
```

Comment:
This would also apply to twr_node_tdma, and almost every other app. Please update them all.